webpack.config.js

```diff
@@ -1,14 +1,43 @@
-{
-  module.exports = {
-    entry: "./lib/index.js",
-    output: {
-      path: __dirname,
-      filename: "./dist/react-dropzone.js",
-      libraryTarget: "var",
-      library: "Dropzone"
+var webpack = require('webpack');
+var path = require('path');
+
+module.exports = {
+	entry: "./src/index.js",
+    devtool: "source-map",
+	output: {
+		path: __dirname,
+		filename: "./dist/react-dropzone.js",
+		libraryTarget: "umd",
+		library: "Dropzone"
+	},
+    module: {
+        loaders: [
+            {
+                include: [
+                    path.resolve(__dirname,"src"),
+                ],
+                test: /\.js$/,
+                loader: 'babel-loader',
+                query: {
+                    presets: ['es2015','react','stage-0'],
```

Comment:
In other words it might be better to go back to just using `.babelrc`, I thought webpack needed the parameters specified in `webpack.config.js` but that does not seem to be the case. I'll update the PR.